CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * [#90](https://github.com/matt-kruse/alexa-app/pull/90): Added [Danger](http://danger.systems), PR linter - [@dblock](https://github.com/dblock).
 * [#91](https://github.com/matt-kruse/alexa-app/pull/91): Fixed "Cannot read property 'new' of undefined" init error - [@fremail](https://github.com/fremail).
 * [#88](https://github.com/matt-kruse/alexa-app/pull/88), [#92](https://github.com/matt-kruse/alexa-app/pull/92): AudioPlayer functionality - [@wschaeferiii](https://github.com/wschaeferiii) and [@fremail](https://github.com/fremail).
+* [#101](https://github.com/matt-kruse/alexa-app/pull/101): Expect request `context` to always be present and not modify that data. [@trayburn](https://github.com/trayburn)
```

Comment:
Also, this didn't exist in 2.3.4 (double check please), so the change should be something like "Added `request.context`", right?